Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/FreightAnimations.cs

```diff
@@ -863,6 +863,22 @@ public bool CheckForMerge(int i)
             }
             return false;
         }
+
+        /// <summary>
+        /// Removes discrete freight animations and containers when AI train is deleted
+        /// </summary>
+
+        public void RemoveDiscreteFreightAnimations(MSTSWagon wagon)
```

Comment:
Shouldn't the function by called RemoveContainers?
Can't you use the Wagon member which is already present in the class?

Reply:
OK, I have updated the code.